Makefile

```
@@ -30,20 +30,29 @@ export CGO_ENABLED:=0
 export GO111MODULE:=on
 export GOPROXY?=https://proxy.golang.org/
 
-all: format test build/operator-sdk
+all: format test build/operator-sdk ## Test and Build the Operator SDK
 
-format:
+format: ## Format the source code
 	$(Q)go fmt $(PKGS)
 
-tidy:
+tidy: ## Update dependencies
 	$(Q)go mod tidy -v
 
```

WDYT about organize them by category?

```suggestion
##@ Application
##@ Developing
##@ Test
```

---

I like the reorganization idea, but I'd prefer not to tackle that in this PR. I think it will make it more difficult to understand what the changes that are being made. After the help gets merged, I'd say we look at other cleanup.